pages/index.tsx

```diff
@@ -1,10 +1,18 @@
+import { useRouter } from "next/dist/client/router";
 import Head from "next/head";
 import RulesBtn from "../components/button/RulesBtn";
 import StartGameBtn from "../components/button/StartGameBtn";
 import Logo from "../components/logo/Logo";
 import styles from "../styles/Home.module.css";
 
 export default function Home() {
+  const router = useRouter();
+
+  const goToTest = (e) => {
+    e.preventDefault();
+    router.push("/test");
+  };
```

Comment:
If the buttons are only used as link, better use `a`. 
If not, I would add `cursor: pointer` to your buttons to indicate, that you can click them.